### ISSUE TITLE
net = torch.load('ShuffleNetV2_0.98.pth') 

### ISSUE THREAD

**Issue by yangyuke001** [MIPT-Oulu/pytorch_bn_fusion#5]
how to use my pre-trained weights ?
example:
net = torch.load('ShuffleNetV2_0.98.pth') 
does no work.
thanks!

**Comment by yangyuke001** [comment]
i got it .
